assets/sass/components/language.scss

```diff
@@ -2,12 +2,15 @@
   background: {
     position: left center;
     repeat: no-repeat;
-    size: 1.5em 1.5em; // 24px 24px
+    size: 1.6em 1em;
```

Comment:
With the `font-size` of this element being `1.6rem` = `16px`, the background is now 25.6 * 16px large. I’d rather not leave the rounding up to the browsers. If you want to go for a width of 25px use `1.5625em` (25/16), if you want 26px use `1.625em` (26/16).